.npmrc

```diff
@@ -1 +1,2 @@
 package-lock=false
+npm install paypal-examples/docs-examples/tree/main/standard-integration
```

Comment:
Hi @AldemirPereira, this is an interesting idea to automatically "npm install" dependencies for the different integrations. Does this work for you? I tested locally and could not get "npm install" to run when set in the .npmrc file.